Documentation/gettingstarted/kind.rst

```diff
@@ -44,7 +45,20 @@ Install Cilium release via Helm:
       --set global.externalIPs.enabled=true \\
       --set global.nodePort.enabled=true \\
       --set global.hostPort.enabled=true \\
-      --set global.pullPolicy=IfNotPresent
+      --set global.pullPolicy=IfNotPresent \\
+      --set config.ipam=kubernetes
```

Comment:
@Rolinh what about
```
--set ipam.operator.clusterPoolIPv4PodCIDR=10.244.0.0/16
```

instead?

Reply:
Would you mind explaining the difference? With `config.ipam=kubernetes`, the range was picked up correctly by Cilium.

Reply:
For testing purposes, I recreated a cluster and tried your method. Unfortunately, I end up with a cluster with broken connectivity. 
```
$ kubectl get nodes kind-worker2 -o yaml
apiVersion: v1
kind: Node
metadata:
  annotations:
    io.cilium.network.ipv4-cilium-host: 10.0.3.220
    io.cilium.network.ipv4-health-ip: 10.0.3.4
    io.cilium.network.ipv4-pod-cidr: 10.0.3.0/24
...
spec:
  podCIDR: 10.244.1.0/24
  podCIDRs:
  - 10.244.1.0/24
```
I have pods in the IP range that conflict with my network instead of `10.244.0.0/16`:
```
$ kubectl get pods -A -o wide | grep 10\.
kube-system          coredns-66bff467f8-4wl8t                     1/1     Running            0          9m39s   10.0.1.109   kind-worker          <none>           <none>
kube-system          coredns-66bff467f8-p2st8                     1/1     Running            0          9m39s   10.0.0.100   kind-control-plane   <none>           <none>
kube-system          hubble-relay-5684848cc8-7jhl9                0/1     ImagePullBackOff   0          7m26s   10.0.2.152   kind-worker3         <none>           <none>
kube-system          hubble-ui-54c6bc4cdc-vnhjb                   0/1     ImagePullBackOff   0          7m26s   10.0.1.196   kind-worker          <none>           <none>
local-path-storage   local-path-provisioner-bd4bb6b75-n62cq       1/1     Running            0          9m39s   10.0.1.203   kind-worker          <none>           <none>
```